stdlib/public/Python/Python.swift

```diff
@@ -308,7 +312,7 @@ public struct ThrowingPythonObject {
       try throwPythonErrorIfPresent()
       throw PythonError.invalidCall(base)
     }
-    return PythonObject(owning: result)
+    return PythonObject(consuming: result)
```

Comment:
Related comment: do you and @dan-zheng think “moving:” might be better than “consuming:” here?

Reply:
Nope. Moving has a well-defined meaning in c++ that does not apply here. This is how you would move from a smart-pointer, but it is not a smart-pointer.

Reply:
I don't have a preference. :)